Moved warning note one step up and fixed text

sandervandevelde opened the pull request:
To prevent a user from already executing a command before reading the warning note, it can be moved one step up. And the text had to point to the correct (next) command.

azurecla commented:
Hi __@mmccrory__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@mmccrory There's a merge conflict on this request.  Could you please resolve the merge conflict?  If you need any help feel free to ask me.

deneha commented:
Fixed merge conflict and incorporated edits into https://github.com/Azure/azure-content-pr/pull/12260

sandervandevelde committed:
Moved warning note one step up

Moved warning note one step up, other wise the step could already be executed before the note is read

sandervandevelde committed:
Altered the text in moved warning note

Text in the warning note (which is moved up one step) is now speaking about 'The following command'
